docs: add say_stream notes to the sending messages page (#1463)
slackapi/bolt-python: docs/english/concepts/message-sending.md

## Streaming messages {#streaming-messages}

You can have your app's messages stream in to replicate conventional agent behavior. This is done through three Web API methods:

* [`chat_startStream`](/reference/methods/chat.startStream)
* [`chat_appendStream`](/reference/methods/chat.appendStream)

> * [`chat_stopStream`](/reference/methods/chat.stopStream)

Reviewer: nit(non-blocking):  Showing the 3 Web API methods to manually stream messages at the very top is a little confusing.  Also, L54 mentions that `say_stream` uses this API under-the-hood and there's a nice callout at the bottom explaining how to manually use the Web API Streaming methods.

Could this sentence be merged with the next paragraph? For example:

> You can have your app's messages stream in to replicate conventional agent behaviour. Bolt for Python provides a `say_stream` utility is available on `app.event` and `app.message` listeners.

> Bolt for Python provides a `say_stream` listener argument available on `app.event` and `app.message` listeners. 

Reviewer 2: super nice ⭐

> Bolt for Python provides a `say_stream` listener argument available on `app.event` and `app.message` listeners. 

Reviewer: suggestion: The current sentence is correct, but "argument" is often thought of as a simple variable or object. We use the term "utility" below, so maybe can clarify that `say_stream` is a utility provided as a listener argument.

```suggestion
Bolt for Python provides a `say_stream` utility as a listener argument for `app.event` and `app.message` listeners. 
```

The `say_stream` utility streamlines calling the Python Slack SDK's [`WebClient.chat_stream`](https://docs.slack.dev/tools/python-slack-sdk/reference/web/client.html#slack_sdk.web.client.WebClient.chat_stream) helper utility by sourcing parameter values from the relevant event payload.

| Parameter | Value |
|---|---| 
| `channel_id` | Sourced from the event payload.
| `thread_ts` | Sourced from the event payload. Falls back to the `ts` value if available.
| `recipient_team_id` | Sourced from the event `team_id` (`enterprise_id` if the app is installed on an org).
| `recipient_user_id` | Sourced from the `user_id` of the event.

> If neither a `channel_id` or `thread_ts` can be sourced, then the utility will merely be `None`.

Reviewer 2: nit: i feel like this can be clearer
```suggestion
If neither a `channel_id` or `thread_ts` can be sourced, then the utility will just be `None`.
```
